webtau-rest-groovy/src/test/groovy/com/twosigma/webtau/http/HttpTest.groovy

```diff
@@ -236,6 +243,52 @@ class HttpTest {
         assert transformed == []
     }
 
+    @Test
+    void "equality matcher"() {
+        http.get("/end-point") {
+            id.shouldNot == 0
```

Comment:
Is it worth adding dates to these tests?

Reply:
I wonder if a separate dates section should be there. I didn't since dates were in a separate PR. This PR is dates agnostic. I will keep updating docs in a separate PR.

Reply:
OK.